tests/modules/nf-core/vcfanno/test.yml

```diff
@@ -1,23 +1,15 @@
 - name: vcfanno test_vcfanno
-  command: nextflow run ./tests/modules/nf-core/vcfanno -entry test_vcfanno -c ./tests/config/nextflow.config  -c ./tests/modules/nf-core/vcfanno/nextflow.config
+  command: nextflow run ./tests/modules/nf-core/vcfanno -entry test_vcfanno -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/vcfanno/nextflow.config
   tags:
     - vcfanno
   files:
-    - path: output/untar/vcfanno_grch38_module_test/exac.vcf.gz
-      md5sum: eba4b55d81148d9c316cda68a5c0ad6d
-    - path: output/untar/vcfanno_grch38_module_test/exac.vcf.gz.tbi
```

Comment:
Was the .tbi output included by mistake previously?

Reply:
Neither of those files should've been included, as they're inputs